crates/fuel-core/src/service/sub_services.rs

```diff
@@ -288,6 +293,7 @@ pub fn init_sub_services(
         chain_id,
         config.da_compression.clone(),
         config.continue_on_error,
+        *base_asset_id,
```

Comment:
```suggestion
        base_asset_id,
```

Reply:
Updated in https://github.com/FuelLabs/fuel-core/commit/a915976cc879aa058264fe0fd4a24168eddd8cac